samples/stock/demo/eth-btc-areachart/demo.js

```diff
@@ -0,0 +1,160 @@
+Highcharts.setOptions({
+    lang: {
+        rangeSelectorZoom: ''
+    }
+});
+
+Highcharts.stockChart('container', {
+
+    chart: {
+        backgroundColor: '#000000'
+    },
+
+    title: {
+        text: 'ETHBTC',
+        align: 'left',
+        y: 30,
+        style: {
+            color: '#b0abab',
+            fontSize: '2em'
+        }
+    },
+
+    xAxis: {
+        lineWidth: 0,
+        labels: {
+            style: {
+                color: '#b0abab'
+            }
+        }
+    },
+
+    scrollbar: {
+        barBorderRadius: 8,
+        barBackgroundColor: '#5f5959',
+        height: 8,
+        margin: 0,
+        trackBorderColor: '#5f5959',
+        trackBorderRadius: 8
+    },
+
+    exporting: {
+        buttons: {
+            contextButton: {
+                symbolStroke: '#d9d7d7',
+                theme: {
+                    fill: '#000000'
+                }
+            }
+        }
+    },
+
+    yAxis: {
+        gridLineWidth: 0,
+        max: 0.07,
+        min: 0.045,
+        offset: 30,
+        labels: {
+            style: {
+                color: '#b0abab'
+            }
+        },
+        accessibility: {
+            description: 'price in Bitcoin'
+        }
+    },
+
+    navigator: {
+        enabled: false
+    },
+
+    rangeSelector: {
+        buttonPosition: {
+            align: 'right',
+            x: -30,
+            y: -40
+        },
+        buttonSpacing: 10,
+        buttonTheme: {
+            fill: 'none',
+            stroke: 'none',
+            'stroke-width': 0,
+            r: 8,
+            style: {
+                color: '#d9d7d7',
+                fontWeight: 'bold',
+                fontSize: '1em'
+            },
+            states: {
+                select: {
```

Comment:
Could you also update `hover` state? A bit unreadable at this moment:
![image](https://github.com/highcharts/highcharts/assets/1453926/9ef3a3a0-2e2e-47d6-94d1-5582489a8afb)